app/models/miq_group.rb

```diff
@@ -106,7 +103,7 @@ def self.get_ldap_groups_by_user(user, bind_dn, bind_pwd)
     user_obj = ldap.get_user_object(ldap.normalize(ldap.fqusername(username)))
     raise _("Unable to find user %{user_name} in directory") % {:user_name => username} if user_obj.nil?
 
-    ldap.get_memberships(user_obj, auth[:group_memberships_max_depth])
+    ldap.get_memberships(user_obj, ::Settings.authentication.group_memberships_max_depth)
```

Comment:
This needs the default value now...

```ruby
ldap.get_memberships(user_obj, ::Settings.authentication.group_memberships_max_depth ||  User::DEFAULT_GROUP_MEMBERSHIPS_MAX_DEPTH)
```

Or, we should make it the default in the settings.yml and remove the constant.